pkg/broker/filter/filter_handler.go

```diff
@@ -275,6 +276,7 @@ func (h *Handler) handleDispatchToDLSRequest(ctx context.Context, trigger *event
 		trigger:     trigger.Name,
 		broker:      trigger.Spec.Broker,
 		requestType: "dls_forward",
+		requestScheme: request.URL.Scheme,
```

Comment:
the value of request.URL is empty on the Server side, see https://github.com/golang/go/issues/28940, we need to get the https value differently based on whether the server is TLS or not

Reply:
It looks like https://github.com/golang/go/issues/28940#issuecomment-441498136 may be a good workaround in the case where TLS is permissive (and we have both http and https)

Reply:
@sadath-12 I don't think this is resolved yet, as `request.URL` will always be empty on the server side, and this is on the server side. Currently, all this is doing is it will always report `http` even if the request should be reported as `https`